src/test/java/com/mentoring/DbPocTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mentoring;
+
+import com.mentoring.framework.utils.Database;
+import com.mentoring.framework.utils.DbSelects;
+import lombok.extern.slf4j.Slf4j;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.sql.SQLException;
+
+@Slf4j
+public class DbPocTest {
```

Comment:
Just an FYI: You can remove this class later on when we're going to use db scripts in real tests, but let's just leave it here until.